### PR TITLE
Omega Worm Balance Changes/Quality of Life Updates

### PR DESCRIPTION
**Pull request by SirChuckOfTheChuckles**
What was changed:

- Omega Worm is now able to create drones quickly (6 seconds, as of pull request). Does NOT require Ouroboros Strain (the upgrade required for creating combat units). Works with Twin Drones upgrade:

https://github.com/user-attachments/assets/432bad8b-a9e8-44f6-b626-5b40d02bd134

(Note: Different rally points can now be set for workers and combat units):

https://github.com/user-attachments/assets/d5f8c33c-cd67-4b7d-bdd5-04bc9708336b

Worker count is now displayed above Omega Worms:
image

A Nydus Network can now make a Nydus Worm and Omega Worm simultaneously; worms can now be canceled mid-construction, and no longer have armor disabled while emerging.

https://github.com/user-attachments/assets/05b13deb-a7db-4c7b-92e7-6ebac7af15fa

Goal of these changes were to make Omega Worms feel more useful as a base of operations pre-Ouroboros Strain, and to remove some frustration encountered when playtesting them.

### CODE REVIEW

File: Mods/ArchipelagoPlayer.SC2Mod/Base.SC2Data/GameData/ActorData.xml [hunk 1]
```diff
@@ -1062,7 +1062,7 @@
         <SoundArray index="Yes" value="NydusWorm_What"/>
         <SoundArray index="Construction" value="Zerg_BuildingMorphWhat"/>
         <UnitFlags index="ShowBlobShadow" value="0"/>
-        <UnitIcon value="AP\Assets\Textures\OmegaNydusDDSIcon.dds"/>
+        <UnitIcon value="Assets\Textures\talent-kerrigan-level08-greaternydusworm.dds"/>
```
Comment (MatthewMarinets) [inline Mods/ArchipelagoPlayer.SC2Mod/Base.SC2Data/GameData/ActorData.xml:1065]: The old icon path also shows up in 4 places in ButtonData.xml

Comment (SirChuckOfTheChuckles) [reply Mods/ArchipelagoPlayer.SC2Mod/Base.SC2Data/GameData/ActorData.xml:1065]: Thank you, those have been changed as well. thought it was just the actor data with that information

Comment (MatthewMarinets) [reply Mods/ArchipelagoPlayer.SC2Mod/Base.SC2Data/GameData/ActorData.xml:1065]: The other icons you added, ArmyPanelIcon and MinimapIcon, probably also exist in the game files, I just don't have their exact filenames on-hand.

**Comment by SirChuckOfTheChuckles** [comment]
Made other small changes:
Changed the hotkey on salvage to match the salvage keys on bunkers
Creating a creep tumor will no longer be interrupted once you start to train a unit
Changed the icon of Omega Worm to look more natural

image

File: Mods/ArchipelagoPlayer.SC2Mod/Base.SC2Data/GameData/UnitData.xml [hunk 2]
```diff
@@ -9599,6 +9600,7 @@
             <LayoutButtons Face="AP_BirthRaptor" Type="AbilCmd" AbilCmd="AP_OmegaTrain,Train3" Row="0" Column="0"/>
             <LayoutButtons Face="AP_BirthSwarmling" Type="AbilCmd" AbilCmd="AP_OmegaTrain,Train2" Row="0" Column="0"/>
             <LayoutButtons Face="AP_BirthZergling" Type="AbilCmd" AbilCmd="AP_OmegaTrain,Train1" Row="0" Column="0"/>
+            <LayoutButtons Face="AP_BirthDrone" Type="AbilCmd" AbilCmd="AP_OmegaTrain,Train4" Row="2" Column="1"/>
             <LayoutButtons Face="AP_BirthCorpser" Type="AbilCmd" AbilCmd="AP_OmegaTrain,Train6" Row="0" Column="1"/>
             <LayoutButtons Face="AP_BirthVile" Type="AbilCmd" AbilCmd="AP_OmegaTrain,Train7" Row="0" Column="1"/>
             <LayoutButtons Face="AP_BirthRoach" Type="AbilCmd" AbilCmd="AP_OmegaTrain,Train5" Row="0" Column="1"/>
```
Comment (Ziktofel) [inline Mods/ArchipelagoPlayer.SC2Mod/Base.SC2Data/GameData/UnitData.xml:9606]: I'd put Drone into first row before lings. Cancel Last should be at bottom-right slot

**Review by Ziktofel — CHANGES_REQUESTED** [review]
Bug: Rallying with Y-key (or the button) doesn't work (Try rallying combat units onto a mineral patch)

File: Mods/ArchipelagoPlayer.SC2Mod/Base.SC2Data/GameData/UnitData.xml (hunk 2 above)
Comment (SirChuckOfTheChuckles) [reply Mods/ArchipelagoPlayer.SC2Mod/Base.SC2Data/GameData/UnitData.xml]: https://github.com/user-attachments/assets/cbaae1b1-acf1-455e-9505-5ed8a2fbc93e

Made requested changes to to command card, and fixed rallying bug.
(Also made a slight nerf to resource efficiency in most recent commit)

**Pull request merged**